pkg/cmd/cli/describe/printer.go

```diff
@@ -80,7 +82,41 @@ func NewHumanReadablePrinter(noHeaders bool) *kctl.HumanReadablePrinter {
 	return p
 }
 
-func printParameters(t *templateapi.Template, w io.Writer) error {
+const templateDescriptionLen = 80
```

Comment:
Just wanted to cap it at something arbitrary.  I'm not eager to go back down the auto-adjusting width path again, but descriptions will probably be several hundred characters for our ootb ones.  We need a better listing function that can show full descriptions (like rhc help commands or rhc list cartridges)